southpark-incremental-indexing/README.restful.md

```diff
@@ -0,0 +1,259 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Jina ❤️ Docker and RESTful APIs](#jina--docker-and-restful-apis)
+  - [Run the Docker](#run-the-docker)
+  - [Index and Search By Yourself](#index-and-search-by-yourself)
+  - [Next Steps](#next-steps)
+  - [Documentation](#documentation)
+  - [Stay tuned](#stay-tuned)
+  - [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Jina ❤️ Docker and RESTful APIs
```

Comment:
Do we really need `README.restful.md`? I feel like that is not the point of this demo.

Reply:
I agree. Removed

Reply:
I think we should NOT c&p the whole South Park README. We should:

1. Say "do the South Park example first"
2. Explain "now you have the South Park code, here's how you modify it for incremental indexing"

This 
- makes maintenance easier
- reduces workload
- in case someone has done south park already and wants to add incremental indexing, they won't need to hunt around WHERE that section is in this new README

Reply:
As we move forwards into restructuring the tutorials, we'll break them down into:

- fundamental unit (e.g. south park search) (i.e. the most basic Jina thing you can program that does sth useful)
- fundamental unit + 1 thing - e.g. south park + incremental indexing
- fundamental unit + 1 different thing - e.g. south park + QueryLang

Since each of these would have different pods and flows, I'd suggest putting them in their own folders (for now), prefixed with `southpark_` (or whatever)

Reply:
Yes, that's how I've done the main README.md

Max's comment was about the .restful

Reply:
D'oh. Hang on, I'll check that too

Reply:
Yeah, I vote for removing the restful README